server/server.go

```diff
@@ -74,7 +74,7 @@ func NewProtocolServer(certFile, keyFile string, logger log.Logger) (*Server, er
 			grpc.MaxConcurrentStreams(math.MaxUint32),
 			grpc.StreamInterceptor(grpc_prometheus.StreamServerInterceptor),
 		),
-		logger: logger,
+		logger:         logger,
```

Comment:
This is minor, but I think it would look nicer without the extra spaces, or, if you wish to have all arguments lined up, I believe lines 72-76 (starting with `grpc.`) should be indented as well.

Reply:
That was actually done by gofmt, so I would leave as it is, otherwise it will change it next time again.